### ISSUE TITLE
[Improvement]: Change `bal add` generated code to adhere to best practices

### ISSUE THREAD

**Issue by gayaldassanayake** [ballerina-platform/ballerina-lang#41775]
### Description

The current generated code when we add a module using `bal add` looks like,

```ballerina
public function hello(string name) returns string {
    if !(name is "") {
        return "Hello, " + name;
    }
    return "Hello, World!";
}
```

### Describe your problem(s)

How ever it is not the best practice to use a NOT to negate the `name is ""`.

### Describe your solution(s)

Instead, we should use `if name !is ""` there.

### Related area

-> Compilation

**Comment by lwpamihiranga** [comment]
Hello, Can I pick this up?

**Comment by ShammiL** [comment]
Replaced the existing generated code with the below as passing a nil value instead of an empty string is encouraged in Ballerina. 
```ballerina
public function hello(string? name) returns string {
    if name !is () {
        return "Hello, " + name;
    }
    return "Hello, World!";
}
```
Fixed in #42136
